### DISCUSSION TITLE
ロゴがほしい！

### DISCUSSION THREAD

**Discussion by kagomen** [first-contributions-ja/first-contributions-ja.github.io#93] [12: 新機能の提案]
どうせならファビコンを設定したい！
ロゴが欲しい！です！

もし作りたい方がいなければ作ります！
何か良さげなアイデアがあればお願いします……！

モチーフになりそうなもの
- 日本
- 赤と白
- PC
- 参加
- はじめの一歩
- GitHub

**Comment by kazzyfrog** [comment]
ロゴ欲しいですね！
ロゴが決まれば、なんとなくの全体的な雰囲気とか、プロジェクトの呼び方とかも決まって、とんとん進みそうな気がしてます！

**Reply by kagomen** [reply]
プロジェクトネームも悩ましいですね……！

**Comment by pss-aileen** [comment]
つ、作りたいです！よければ...！！

モチーフなどは
- 初心者マーク
- 種、芽
- ひよこ
- 白帯（空手の初心者帯）
- 真っ白なノート
- 足マーク？

**Reply by pss-aileen** [reply]
このログよく考えられてるなぁと思う

https://github.com/firstcontributions-bot
![スクリーンショット 2024-03-22 1 30 27](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/140990100/542e10bc-f844-495d-9c08-de6a5fc9e4a4)
first pull request

**Reply by kagomen** [reply]
ではお願いします……✨
急ぎではないと思うので、時間のある時にでも🙏🌟

**Reply by kagomen** [reply]
自分もなんかモチーフとか思いついたら書き込んでいきます！

**Reply by kazzyfrog** [reply]
お願いします！😎
自分も良さげなプロジェクト名とか、モチーフとか、ちょくちょく考えたりしているものの、全く浮かばないっす。。

**Comment by pss-aileen** [comment]
あと、どう設定するかはわかりませんが

[リポジトリ](https://github.com/topics/beginners)

こういった画面で表示されるサムネイル画像的なものもあれば！

![スクリーンショット 2024-03-22 1 45 37](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/140990100/99838abe-03f5-4427-8f58-915de8e2bcbe)

**Reply by kagomen** [reply]
これってREADMEのトップに表示させてる感じなんですかね❓
GitHubのデコり方（？）調べてみます！

**Reply by kazzyfrog** [reply]
確か、GithubリポジトリのデフォルトのOGP画像を変更するのがあった気がします！
[これかな](https://zenn.dev/ryo_kawamata/articles/introduction-socialify)

あれ、普通にリポジトリの設定でできるのかも
https://blog.kondoumh.com/entry/2020/04/09/201454
> リポジトリ Settings の Social preview に画像をアップロードするだけ
